websites/0-9/4anime/presence.ts

```diff
@@ -73,18 +51,26 @@ presence.on("UpdateData", async () => {
     presenceData.startTimestamp = timestamps[0];
     presenceData.endTimestamp = timestamps[1];
 
-    presence.setTrayTitle(video.paused ? "" : videoTitle);
+    presence.setTrayTitle(video.paused ? "" : seriesTitle);
 
-    presenceData.details = videoTitle;
-    presenceData.state = seasonepisode.textContent;
+    presenceData.buttons = [
+      {
+        label: "Episode",
+        url: document.baseURI
+      },
+      {
+        label: "Series",
+        url: series.getAttribute("href")
+      }
+    ];
```

Comment:
This is supposed to direct to watch episode etc? So maybe describe it better like "Watch Episode", "View Series" or something